lib/stripe/token.ex

```diff
@@ -68,6 +69,21 @@ defmodule Stripe.Token do
   end
 
   @doc """
+  Create a token.
+
+  WARNING : This function is mainly for testing purposes only, do not use
+  in production, use the Stripe.js library on the client device instead.
+
+  You MUST pass in your Publishable Key in `opts` :
+
+      Stripe.Token.create(%{card: card_map}, api_key: "STRIPE_PUBLISHABLE_KEY")
```

Comment:
That was a false assumption of mine, since it has to be with the publishable key on the client side, I supposed we needed it too on direct API requests for creating a token (it works with both, though). In the API Docs they are giving an exemple with a secret key as a parameter.
https://stripe.com/docs/api#create_card_token

I will remove the requirement for the publishable key as an argument in the doc section.